### ISSUE TITLE
Move wiki articles into sphinx documentation

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#2920]
As discussed in earlier meeting.

Advantages

- We can make navigation
- Documentation will follow suit with versioning
- Layout will be easier and more powerful than on Github's wiki
- Macros for automatic screenshots etc. in the pipeline

**Comment by benjaoming** [comment]
@aronasorman I think I might have volunteered for this one :)

**Comment by benjaoming** [comment]
This can still safely be implemented for 0.15 it doesn't touch the codebase really :)

**Comment by rtibbles** [comment]
True, but as it is unlikely to happen within the timeline of 0.15, let's punt it for now.

**Comment by benjaoming** [comment]
You've gone punting hunting berzerk! :)

But yeah, let's see what we can do here. Some things might be done, but it doesn't really have to be tracked in an issue anyways. This one can be for the final death of the GH wiki.

**Comment by rtibbles** [comment]
It seems our procrastination knows no bounds.

**Comment by benjaoming** [comment]
indeed!

**Comment by benjaoming** [comment]
Why doesn't Github have a "Procrastinate this issue" button right next to "Close and comment" ?

**Comment by radinamatic** [comment]
Not sure if this is ever going to be "fixed", in the scope of KA Lite lifetime, but I'm jumping onto procrastination train for now and punting it... :stuck_out_tongue:
